cli/src/commands/util/exec.rs

```diff
@@ -66,27 +66,55 @@ use crate::ui::Ui;
 pub(crate) struct UtilExecArgs {
     /// External command to execute
     command: String,
+
     /// Arguments to pass to the external command
     #[arg(value_hint = clap::ValueHint::FilePath)]
     args: Vec<String>,
+
+    // If set, command is treated as a path relative to the workspace root.
+    #[arg(long)]
+    relative_command: bool,
+
+    // If set, rather than attempting to wrap the process, tries to act more
+    // like execv (match return code and don't print anything).
+    #[arg(long)]
+    transparent: bool,
```

Comment:
I would certainly split the `transparent` argument into a separate commit, and maybe even a separate PR.

I wish that this was the default behavior, at least as far as the exit code is concerned, but I'm not sure whether that's feasible.